frontend/src/components/Home/SidePaneView.js

```diff
@@ -43,6 +50,20 @@ export const SidePaneView = () => {
             dispatch(fetchJobsData(startDate, endDate))
     }, [startDate, endDate, dispatch])
 
+    useEffect(() => {
+        let buildParams = ''
+        if(ciSystem !== '') buildParams += `&ciSystem=${ciSystem}`
+        if(platform !== '') buildParams += `&platform=${platform}`
+        if(benchmark !== '') buildParams += `&benchmark=${benchmark}`
+        if(version !== '') buildParams += `&version=${version}`
+        if(workerCount !== '') buildParams += `&workerCount=${workerCount}`
+        if(networkType !== '') buildParams += `&networkType=${networkType}`
+        if(startDate !== '') buildParams += `&startDate=${startDate}`
+        if(endDate !== '') buildParams += `&endDate=${endDate}`
+        history.push(`/optimize_reports?${buildParams.substring(1)}`, { replace: true });
```

Comment:
Why are we replacing the latest entry in the browser history with this new entry?

Reply:
`replace=true` will replace the current URL with a modified URL.